api/pkg/transformer/server/server.go

```diff
@@ -295,6 +300,19 @@ func (s *Server) Run() {
 	}
 }
 
+func recoveryHandler(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		defer func() {
+			if err := recover(); err != nil {
+				debug.PrintStack()
+				w.WriteHeader(http.StatusInternalServerError)
```

Comment:
Can we return error message to the caller?

```
response.NewError(http.StatusInternalServerError, errors.Wrapf(err, "panic")).Write(w)
```

Reply:
I think `w.WriteHeader(http.StatusInternalServerError)` should be removed as `response.NewError(http.StatusInternalServerError, fmt.Errorf("panic: %v", err)).Write(w)` already write the status code.